Config.lua

```diff
@@ -57,21 +57,29 @@ local function createOptionsForClass(class)
 end
 
 local function setFilterValue(info, val)
-    local filter = filterValues[info]
```

Comment:
these setter & getter are used for setting the CLEU filter value. for setting a bool value in config, i would provide own set & get methods for your toggle

Reply:
Fixed.